src/parsing/binast-serialize-visitor.h

```diff
@@ -533,7 +533,16 @@ inline void BinAstSerializeVisitor::ToDoBinAst(AstNode* node) {
 }
 
 inline void BinAstSerializeVisitor::VisitFunctionLiteral(FunctionLiteral* function_literal) {
+  size_t start = byte_data_.size();
+
   SerializeAstNodeHeader(function_literal);
+
+  SerializeUint32(static_cast<int>(start));
```

Comment:
Maybe use `uint32_t` instead of `int` here since `size_t` is unsigned, and add a `DCHECK(start <= UINT_MAX)` just to be a little safer.